lib/rack/protection/authenticity_token.rb

```diff
@@ -20,15 +22,91 @@ class AuthenticityToken < Base
       default_options :authenticity_param => 'authenticity_token',
                       :allow_if => nil
 
+      TOKEN_LENGTH = 32
```

Comment:
This should be default option, which can be configurable, call it `:authenticity_token_length`